### DISCUSSION TITLE
Highlight path to container (eg. highlight tab of current container in a split layout)

### DISCUSSION THREAD

**Discussion by lefuturiste** [i3/i3#5180] [Q&A] [answered]
One of the most useful features I use from i3wm is the ability to have a different layout inside of a parent layout. Like for example in a workspace having a tabbed layout and then multiple horizontal and vertical split layouts in each tab.
The issue that I have is that I find difficult to known is which top level tab I'm currently focused in.
What I would like, is some option to differentiate which tab the container I'm currently focused is in.
In general being able to spot the path it take to go from the root container to the leaf container (highlight each node container to get to the leaf).
I don't know if there is a way to do that in i3wm but if not, I would be happy to contribute to this project for this very specific feature.

Thanks for stopping by.

**Comment by Airblader** [comment, marked as answer]
That's the "focused_inactive" color class.

https://i3wm.org/docs/userguide.html#client_colors

**Reply by lefuturiste** [reply]
Thanks for this I clue, I didn't though about color class. 
I ended up using the `focused_tab_title` because the `focused_inactive` class also highlight neighboring windows in the same tab.
It seems to work fine now. But now I'm noticing that it's quite hard to make sense of the tab title when every window is the same shell program. Probably I need some renaming feature so that I can rename the current tab I am on. I don't know if this is possible.
Thanks again.

**Reply by Airblader** [reply]
You can use `title_format` for that.
